reference/api/indexes.md

```diff
@@ -162,7 +201,13 @@ You can use the response's `taskUid` to [track the status of your request](/refe
 
 <RouteHighlighter method="DELETE" route="/indexes/{index_uid}"/>
 
-Delete an [index](/learn/core_concepts/indexes.md).  The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+Delete an index.
+
+### Path parameters
+
+| Name              | Type   | Description                                                               |
+| :---------------- | :----- | :------------------------------------------------------------------------ |
+| **`index_uid`** * | String | [`uid`](/learn/core_concepts/indexes.md#index-uid) of the requested index |
 
 This is an asynchronous task. [You can read more about asynchronous operations in our dedicated guide.](/learn/advanced/asynchronous_operations.md)
```

Comment:
I think we should only mention this explicitly on certain endpoints, such as create index and add documents.

```suggestion
```